include/mm/slab.h

```diff
@@ -59,6 +59,8 @@ typedef enum slab_size slab_size_t;
 
 #define SLAB_SIZE_FULL_MASK ((SLAB_SIZE_MAX << 1) - 1)
 
+#define MAX_SLAB_ALLOC_COUNT PAGE_SIZE/SLAB_SIZE_MIN
```

Comment:
Is it always PAGE_SIZE we will use for underlying allocation? Global meta slabs use PAGE_SIZE_2M for example. Make make it a macro with page size as argument?

nit: might make sense to add build bug check for the modulo division always returning 0.

Reply:
I changed this code in subsequent PR. 
There won't be any PAGE_SIZE_2M.
All allocations will be 4K and we won't run out of memory unless we run of frames.

Reply:
I removed it's usage in this commit but will be used in next PR.